go.mod

```diff
@@ -1,5 +1,5 @@
 module foxygo.at/evy
 
-go 1.20
+go 1.21
```

Comment:
:pineapple: Did you see the thread on #modules on gopher slack? Apparently the tooling now writes "1.21.0" here which causes parsing problems with < go 1.21.0 (older tool chains do not expect a patch version here and emit a parse error instead of an error about the version being too high).

What I'm saying is I'm not 100% sure this is correct, but I did a little bit of playing around and it seems to be ok to use this form still. So this comment is really just a heads-up in case you see any weirdness here later.

Reply:
Nope completely missed it! Thanks